src/nrtk_explorer/app/transforms.py

```diff
@@ -299,18 +323,17 @@ def update_model_result(self, image_ids, feature_extraction_model):
             result_id = image_id_to_result(image_id)
             self.state[result_id] = self.context["annotations"].get(image_id, [])
 
-    def on_image_hovered(self, index):
-        self.state.hovered_id = index
+    def on_image_hovered(self, id):
+        self.state.hovered_id = id
 
     def set_on_hover(self, fn):
         self._on_hover_fn = fn
 
     def on_hover(self, hover_event):
-        id_ = int(hover_event["id"])
-        is_transformation = bool(hover_event["isTransformation"])
```

Comment:
Why getting rid of the is_transformation boolean parameter? Here and in image_list.py

Reply:
Because `is_transformation` is derived from the image id.  ImageDetector.vue does not change behavior based on that parameter.